LibraryComponents/Classes/AndesCoachmark/Interactors/AndesCoachMarkHighlightInteractor.swift

```diff
@@ -36,14 +37,22 @@ class AndesCoachMarkHighlightInteractor {
         let margin = self.margin + 4
         let rectConverted = view.convert(view.bounds, to: overlayView)
 
+        //If the highlightRect dosen't fit in bodyView, it will be cutted
+        //Horizontal excess
+        let rectConvertedCuttedX = max(rectConverted.minX, margin)
+        var rectConvertedCuttedWidth = rectConverted.width
+        if rectConverted.minX < margin { rectConvertedCuttedWidth -= (margin - rectConverted.minX) }
+        if rectConverted.maxX + margin > overlayView.bounds.width { rectConvertedCuttedWidth -= (margin + rectConverted.maxX - overlayView.bounds.width) }
+
+        //Vertical excess
         if !bodyViewBounds.contains(rectConverted.insetBy(dx: bodyViewBounds.minX, dy: -margin)) {
```

Comment:
Aca hay un problema en el calculo por como se estaba haciendo. Al mirar el `.contains` de `CGRect` te considera la direccion X para decidir si esta incluido o no, pero en este caso solo nos interesa saber la vertical.

```suggestion
        if !isContainedVertically(rectConverted.insetBy(dx: 0, dy: -margin), in: bodyViewBounds) {
```

Usando el `.contains` hace que el highlight se extienda verticalmente cuando la vista se va para uno de los laterales.

![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 10 43 49](https://user-images.githubusercontent.com/12042857/91727026-7b455b80-eb77-11ea-93d4-98f48f91d05e.png)